crates/bitwarden-vault/src/collection.rs

```diff
@@ -9,6 +9,7 @@ use uuid::Uuid;
 
 use crate::VaultParseError;
 
+/// Encrypted Collection state
```

Comment:
What does `state` imply?

```suggestion
/// Encrypted Collection
```